src/python/grpcio_tests/tests/interop/methods.py

```diff
@@ -44,25 +44,45 @@
 from src.proto.grpc.testing import messages_pb2
 from src.proto.grpc.testing import test_pb2
 
+_INITIAL_METADATA_KEY = "x-grpc-test-echo-initial"
+_TRAILING_METADATA_KEY = "x-grpc-test-echo-trailing-bin"
+
+def _maybe_echo_metadata(servicer_context):
+  """Copies metadata from request to response if it is present."""
+  invocation_metadata = dict(servicer_context.invocation_metadata())
+  if _INITIAL_METADATA_KEY in invocation_metadata:
+    initial_metadatum = (
+        _INITIAL_METADATA_KEY, invocation_metadata[_INITIAL_METADATA_KEY])
+    # Send initial metadata expects a list of tuples
```

Comment:
Drop this incorrect comment; `send_initial_metadata` requires merely [a sequence of metadatums](https://github.com/grpc/grpc/blob/5098508d2d41a116113f7e333c516cd9ef34a943/src/python/grpcio/commands.py#L84).